packages/cosmic-proto/package.json

```diff
@@ -13,50 +13,85 @@
   "bugs": {
     "url": "https://github.com/Agoric/agoric-sdk/issues"
   },
-  "type": "module",
   "exports": {
```

Comment:
IIUC, these `exports` prevent the package from being used as described in the README.  I'd prefer seeing the `./dist/codegen/*` contents moved up to `./`, and all of those names exported as ESM, not CommonJS.  I recognize that this may take quite an effort, but I think it would be worthwhile.

Reply:
I'm okay with specifying all the exports. I don't really see an alternative.

Moving the codegen output up to root, I'm not so sure. It makes a bit of a mess. And it's just to support CJS? I don't think that's a requirement. It's ugly that we have `dist/` in the type imports in this repo, but we can live with that a while.